packages/@react-spectrum/datepicker/src/Input.tsx

```diff
@@ -37,29 +37,32 @@ function Input(props, ref) {
   // not cause a layout shift.
   let [reservePadding, setReservePadding] = useValueEffect(false);
   let onResize = useCallback(() => setReservePadding(function *(reservePadding) {
-    if (reservePadding) {
-      // Try to collapse padding if the content is clipped.
-      if (inputRef.current.scrollWidth > inputRef.current.offsetWidth) {
-        let width = inputRef.current.parentElement.offsetWidth;
-        yield false;
+    if (inputRef.current) {
```

Comment:
The diff is weird, this is just wrapping the old if statement in a inputRef.current check